tools/release_do_nothing.py

```diff
@@ -10,879 +10,868 @@
-) -> None:
-    _break_print("conda-forge updates ...")
+        if not self.is_release_candidate:
+            message += (
+                "- Selecting 'stable' in the version switcher also brings up "
+                f"the {self.git_tag} render."
+            )
+        self.wait_for_done(message)
 
-    if not is_release_candidate:
         message = (
-            "NOTE: after several hours conda-forge automation will create a "
-            "Pull Request against conda-forge/iris-feedstock (via the "
-            "regro-cf-autotick-bot). Quicker to sort it now, manually ..."
+            f"Visit https://scitools-iris.readthedocs.io/en/{self.strings.branch} "
+            "to confirm:\n\n"
+            "- The docs have rendered\n"
+            f"- The version badge in the top left includes: {self.strings.branch} .\n"
+            f"- {self.strings.branch} is NOT available in RTD's version switcher.\n\n"
+            "NOTE: the docs can take several minutes to finish building."
         )
-        _break_print(message)
+        self.wait_for_done(message)
 
-    message = (
-        "Make sure you have a GitHub fork of:\n"
-        "https://github.com/conda-forge/iris-feedstock"
-    )
-    _wait_for_done(message)
+    def check_pypi(self):
+        self.print("PyPI checks ...")
+        self.print("If anything goes wrong, manual steps are in the documentation.")
 
-    message = (
-        "Make sure you have a local clone of your iris-feedstock fork.\n"
-        "`cd` into your clone."
-    )
-    _wait_for_done(message)
+        message = (
+            "Confirm that the following URL is correctly populated:\n"
+            f"https://pypi.org/project/scitools-iris/{self.strings.release}/"
+        )
+        self.wait_for_done(message)
 
-    if is_release_candidate:
         message = (
-            "Visit the conda-forge feedstock branches page:\n"
-            "https://github.com/conda-forge/iris-feedstock/branches"
+            f"Confirm that {self.strings.release} is at the top of this page:\n"
+            "https://pypi.org/project/scitools-iris/#history"
         )
-        _wait_for_done(message)
+        self.wait_for_done(message)
+
+        if self.is_release_candidate:
+            message = (
+                f"Confirm that {self.strings.release} is marked as a "
+                f"pre-release on this page:\n"
+                "https://pypi.org/project/scitools-iris/#history"
+            )
+        else:
+            message = (
+                f"Confirm that {self.strings.release} is the tag shown on the "
+                "scitools-iris PyPI homepage:\n"
+                "https://pypi.org/project/scitools-iris/"
+            )
+        self.wait_for_done(message)
+
+        def validate(sha256_string: str) -> str:
+            valid = True
+            try:
+                _ = int(sha256_string, 16)
+            except ValueError:
+                valid = False
+            valid = valid & len(sha256_string) == 64
```

Comment:
```suggestion
            valid = valid and len(sha256_string) == 64
```
I'm still figuring out what `&` does exactly, I think it does bitwise comparisons, so it's comparing `1` (`TRUE`) to `1000000` (`64`).

Reply:
3e6fd1f99f6e5188711ef28e9f2a7510258ec374

I have tested this